### PR TITLE
Improve UT: add api default test

### PR DESCRIPTION
**Pull request by JaneLiuL**
Add api default test to improve ut
Also finish use manifests/noderesourcetopology/crd.yaml to prepares a CRD

### CODE REVIEW

File: pkg/apis/config/v1beta1/defaults_test.go [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+	"k8s.io/utils/pointer"
+	"strconv"
+	"testing"
+)
+
+func TestSetDefaultsCoschedulingArgs(t *testing.T) {
+	tests := []struct {
+		name   string
+		config *CoschedulingArgs
+		expect *CoschedulingArgs
```
Comment (Huang-Wei) [inline pkg/apis/config/v1beta1/defaults_test.go:33]: Can you merge the top-level tests into one test as sub-tests? Ref: https://github.com/kubernetes/kubernetes/blob/66e1d27a59208f3f90016e0790a13e2602fecdeb/pkg/scheduler/apis/config/v1beta2/defaults_test.go#L456

File: pkg/apis/config/v1beta1/defaults_test.go [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+package v1beta1
+
+import (
+	"github.com/google/go-cmp/cmp"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	schedulerconfig "k8s.io/kube-scheduler/config/v1"
+	"k8s.io/utils/pointer"
+	"strconv"
+	"testing"
```
Comment (Huang-Wei) [inline pkg/apis/config/v1beta1/defaults_test.go:26]: nit: move this up and add an empty line to separate them.

File: pkg/apis/config/v1beta1/defaults_test.go (hunk 1 above)
Comment (JaneLiuL) [reply pkg/apis/config/v1beta1/defaults_test.go:33]: Good idea~ Thanks for your comment ~~ I will update soon~

Comment (JaneLiuL) [reply pkg/apis/config/v1beta1/defaults_test.go]: Fix done~

File: pkg/apis/config/v1beta1/defaults_test.go (hunk 2 above)
Comment (JaneLiuL) [reply pkg/apis/config/v1beta1/defaults_test.go]: fix done~ :)

**Comment by Huang-Wei** [comment]
/lgtm
/approve

Thanks @JaneLiuL !

**Pull request merged**